From 54aab968858bbedfc15b3ff5d457f11b00444312 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Thu, 6 Apr 2017 09:36:11 +0800
Subject: [PATCH] soc: rockchip: fixed compilation error

This adds fixed below errors when compiled rockchip_defconfig without
CONFIG_RK3368_SCPI_PROTOCOL select:

In file included from drivers/clk/rockchip/clk-ddr.c:23:0:
include/soc/rockchip/scpi.h:89:12: warning:
'scpi_sys_set_jtagmux_on_off' defined but not used [-Wunused-function]
error, forbidden warning: scpi.h:89
scripts/Makefile.build:258: recipe for target
'drivers/clk/rockchip/clk-ddr.o' failed
make[3]: *** [drivers/clk/rockchip/clk-ddr.o] Error 1

Change-Id: I5abc184554dcfc3697be82aede8dec27da2fcdd9
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 include/soc/rockchip/scpi.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/soc/rockchip/scpi.h b/include/soc/rockchip/scpi.h
index a2fc901f9cd2..784a9f694af1 100644
--- a/include/soc/rockchip/scpi.h
+++ b/include/soc/rockchip/scpi.h
@@ -86,7 +86,7 @@ static inline int scpi_get_sensor_value(u16 sensor, u32 *val)
 	return -EPERM;
 }
 
-static int scpi_sys_set_jtagmux_on_off(u32 en)
+static inline int scpi_sys_set_jtagmux_on_off(u32 en)
 {
 	return -EPERM;
 }
-- 
2.35.3

